core/controllers/domain_objects_validator.py

```diff
@@ -96,3 +97,14 @@ def validate_collection_change(obj):
     # No explicit call to validate_dict method is necessary, because
     # CollectionChange calls validate method while initialization.
     collection_domain.CollectionChange(obj)
+
+
+def validate_state_dict(state_dict):
+    # type: (Dict[Any, Any]) -> None
```

Comment:
The keys will be of type Text, right? In general, avoid Any as much as possible.

Reply:
Thank you, updated.